app/views/parties/current_members_letters.html.haml

```diff
@@ -0,0 +1,7 @@
+%h1
+  = @party.respond_to?(:partyName) ? @party.partyName : ''
+%ul.people-list
+  - @people.each do |person|
+    %li
+      = person.respond_to?(:personGivenName) ? person.personGivenName : ''
```

Comment:
this has been done. Please see new pull request for branch: aebirim/new-changes-party-controller